incubator/prometheus/templates/_helpers.tpl

```diff
@@ -0,0 +1,10 @@
+{{/*
+Create a default fully qualified app name.
+
+We truncate at 24 chars because some Kubernetes name fields are limited to this
+(by the DNS naming spec).
+*/}}
+{{define "fullname"}}
```

Comment:
I agree, it's boilerplate that could easily be injected as a value or a function we could add. I've created an issue here https://github.com/kubernetes/helm/issues/1186